cmd/cluster/create.go

```diff
@@ -47,14 +54,16 @@ func NewCreateCommand() *cobra.Command {
 	}
 
 	opts := Options{
-		Namespace:          "clusters",
-		Name:               "example",
-		ReleaseImage:       releaseImage,
-		PullSecretFile:     "",
-		AWSCredentialsFile: "",
-		SSHKeyFile:         filepath.Join(os.Getenv("HOME"), ".ssh", "id_rsa.pub"),
-		NodePoolReplicas:   2,
-		Render:             false,
+		Namespace:             "clusters",
+		Name:                  "example",
+		ReleaseImage:          releaseImage,
+		PullSecretFile:        "",
+		AWSCredentialsFile:    "",
+		SSHKeyFile:            filepath.Join(os.Getenv("HOME"), ".ssh", "id_rsa.pub"),
+		NodePoolReplicas:      2,
+		Render:                false,
+		Region:                "us-east-1",
+		WorkerInstanceProfile: "hypershift-worker-profile",
```

Comment:
should probably set all field here, even if they have no default just to make it explicit that we do not default it.  I see `InfraID` and `InfrastructureJSON` missing